azure/functions_worker/bindings/servicebus.py

```diff
@@ -131,6 +131,11 @@ def from_proto(cls, data: protos.TypedData, *,
         elif data_type == 'json':
             body = data.json.encode('utf-8')
 
+        elif data_type is None:
```

Comment:
Can we ever be in a situation where the data is `None` but the `data_type` is not set to `None`? I'm imagining a case where `data` takes a `None` value but the `data_type` is `string`. Is the assumption that cases like this will be handle by `grpc` itself?

Reply:
We can be defensive here, but `None` value in data with `data_type` present seems to me like a functions host bug.

Reply:
We should be fine leaving as it is. `data_type` is assigned whenever `data` is assigned - nothing should be assigned in the case of `data` being `None`